src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -195,6 +195,18 @@ double BulletPhysicsManager::getSceneRestitutionCoefficient() const {
       ->getRestitutionCoefficient();
 }
 
+const std::pair<Magnum::Vector3, Magnum::Vector3>
```

Comment:
Could be. Also could be used to construct a `Magnum::Range3D`. Currently, this just matches the return type format of the Bullet query function. Since it is not exposed to the base `PhysicsManager` API, I left it that way.

Reply:
Structs are sometimes preferred to pair when the type is used in a large scope (especially if it is used in a header file). pairs are preferred for smaller scopes (e.g. a single function) and can result in hard to read code when used in a large scope. It is somewhat analogous to the use of the variable name "i" in a for loop.